src/index.js

```diff
@@ -18,13 +20,13 @@ app.use(express.urlencoded({ extended: false }));
 app.use(cors());
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
-// App routers here
-Route(app);
 // App index
 app.get('/', (req, res) => {
   res.status(200).json({ Message: 'Welcome! This is the NorthStar Barefoot Nomad homepage.' });
 });
 
+app.use('/api/v1/auth', authRouter);
```

Comment:
`/auth` is not common to all routes. Thus, I suggest it be specified in the `user.routes.js` file